### ISSUE TITLE
 libmysqlclient fails on shared x86_64 target, Jessie

### ISSUE THREAD

**Issue by starius** [mxe/mxe#914]
Log https://gist.github.com/starius/4f1246333d5baaa8a3fe
See c0548aa91a77096f7fea07d2bff0e526822b1b75
See #839

**Comment by tonytheodore** [comment]
Skipping the `def` file fixes it, but I don't know why:

```diff
diff --git a/src/libmysqlclient.mk b/src/libmysqlclient.mk
index 104ef7f..7e92abe 100644
--- a/src/libmysqlclient.mk
+++ b/src/libmysqlclient.mk
@@ -36,7 +36,8 @@ define $(PKG)_BUILD
         '$(1)'

     # def file created by cmake creates link errors
-    cp '$(PWD)/src/$(PKG).def' '$(1).build/libmysql/libmysql_exports.def'
+    $(if $(findstring i686-w64-mingw32.shared,$(TARGET)),
+        cp '$(PWD)/src/$(PKG).def' '$(1).build/libmysql/libmysql_exports.def')

     $(MAKE) -C '$(1).build' -j '$(JOBS)' VERBOSE=1
     $(MAKE) -C '$(1).build/include'  -j 1 install VERBOSE=1
```

**Comment by starius** [comment]
Works for me too.

**Comment by starius** [comment]
Fixed in c1cd559
